utils/field.js

```diff
@@ -250,16 +250,29 @@ function prepareFieldForTemplates(entityWithConfig, field, generator) {
     if (field.autoGenerate === undefined) {
       field.autoGenerate = !entityWithConfig.primaryKey.composite && [LONG, UUID].includes(field.fieldType);
     }
+
     if (!field.autoGenerate) {
       field.liquibaseAutoIncrement = false;
       field.jpaGeneratedValue = false;
+      field.autoGenerateByService = false;
+      field.autoGenerateByRepository = false;
+    } else if (entityWithConfig.databaseType !== SQL) {
+      field.liquibaseAutoIncrement = false;
+      field.jpaGeneratedValue = false;
+      field.autoGenerateByService = field.fieldType === UUID;
+      field.autoGenerateByRepository = !field.autoGenerateByService;
```

Comment:
Is this one mandatory?

Reply:
I mean, a `not` is acceptable in a template, the illustration is that you use `f => !f.autoGenerateByRepository` instead of its service counterpart in your PR :-p.
Choose you poison, but keep only one please (if there is a vote, I would choose the `autoGenerateByService`one, as it is the less standard)

Reply:
They are different:
`f => !f.autoGenerateByRepository` should be `f => !f.autoGenerate || f.autoGenerateByService`

If you still think it's better remove one of them, I can do it.

Reply:
Got it!